eventio/event_io_file.py

```diff
@@ -0,0 +1,65 @@
+import struct
+from .object_header import make_ObjectHeader
+from io import BytesIO
+
+class EventIOObject:
```

Comment:
Butbthisba list of the headers oft its subitems, right? Not the object's own item.